packages/health/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt

```diff
@@ -774,7 +774,7 @@ class HealthPlugin(private var channel: MethodChannel? = null) : MethodCallHandl
         }
         healthData.add(
           hashMapOf(
-            "workoutActivityType" to workoutTypeMap.filterValues { it == session.activity }.keys.first(),
+            "workoutActivityType" to (workoutTypeMap.filterValues { it == session.activity }.keys.firstOrNull() ?: "OTHER"),
```

Comment:
Can we use `FitnessActivities.OTHER` instead of introducing a new magic string literal?

Reply:
using `FitnessActivities.OTHER` instead of string literal now